js/okx.js

```diff
@@ -4685,9 +4685,11 @@ module.exports = class okx extends Exchange {
          * @method
          * @name okx#setLeverage
          * @description set the level of leverage for a market
+         * @see https://www.okx.com/docs-v5/en/#rest-api-account-set-leverage
```

Comment:
@Dan-krm, apparently we need a `posSide` for the isolated margin, can we add that requirement?